src/mmw/bundle.sh

```diff
@@ -113,6 +118,7 @@ done
 VAGRANT_COMMAND="cd /opt/app && \
     $ENSURE_DIRS_EXIST && { \
     $COPY_IMAGES_COMMAND &
+    $COPY_SIMPLE_APP_DATA_COMMAND &
```

Comment:
Sorry, my mistake. The command that I was looking for is

```
./scripts/manage.sh collectstatic --noinput
```

It collects all static assets to the Django static directory. See here for details: https://docs.djangoproject.com/en/1.8/ref/contrib/staticfiles/